### DISCUSSION TITLE
firewall rules

### DISCUSSION THREAD

**Discussion by Arbel-arad** [Vaskivskyi/asusrouter#538] [Q&A]
is it possible to add/remove firewall rules?
more specifically i need to change IPv6 firewall rules each time the dynamic prefix changes.

**Comment by omersht** [comment]
I have the exact same issue, have you found any workaround?

**Reply by Arbel-arad** [reply]
> I have the exact same issue, have you found any workaround?

i ended up writing a script to generate iptables files and periodic SSH connections that deploy them
this is a very sketchy solution and i wouldn't recommend it unless you have no other options

you can see the script here (and the device info it reads to generate rules)
https://forgejo.spacetime.technology/arbel/enrollment-inventory/src/branch/master/maps/firewall/makerules.sh
